sdk/core/azure-core-http-okhttp/src/main/java/module-info.java

```diff
@@ -6,6 +6,7 @@
 
     requires okhttp3;
     requires okio;
+    requires kotlin.stdlib;
```

Comment:
We should consider having an okhttp impl that doesn't bring in kotlin

Reply:
Yeah, I was surprised this was needed when OkHttp's `Headers.forEach` was used